CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to KoLmafia
+
+First off, thank you for contributing your time and knowledge. This project
+would not exist if not for volunteers like you!
+
+## Some useful prerequisites (lifted from [Github docs](https://github.com/github/docs/blob/main/CONTRIBUTING.md))
+
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+All work must be done in feature branches (typically within your fork of the
+project), and reviewed by at least one of the core developers through Github's
+Pull Request feature.
+
+We strongly encourage that you provide tests as part of your pull request, as
+this helps us understand the problem at hand and avoid breaking your change in
+the future.
+
+## Guidelines (adapted from [our previous guide](https://kolmafia.us/threads/25364))
+
+(Editor's note: you can mentally replace all instances of "patch" with
+"pull request".)
+
+A successful submit requires social and technical skills.
+
```

Comment:
I would not complain if this were tweaked.  It was written at a time when the social aspects were prominent - at least one dev (me) was ignoring patches from people who were being rude, annoying and acting as if the fact that they did a little work entitled them to have that work incorporated into the repository.  I think the GitHub workflow with reviews is a good step towards reducing the issues caused by personalities.  I would like to encourage discussions "should we do this?  is this a good idea?" before someone writes code and submits a pull request but we could probably rework this.

Reply:
I've rewritten that initial paragraph to incorporate a lot of the things you mentioned. Thoughts?